<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>summarywindow</class>
 <widget class="QMainWindow" name="summarywindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>交易摘要</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1,
                                 stop: 0 #add8e6, stop: 1 #7bb3d9);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #c3e2f0, stop: 1 #87ceeb);
    font-family: &quot;Microsoft YaHei UI&quot;;
    color: #212529;
}
QWidget[class=&quot;QWidget&quot;] {
    background: transparent;
}
</string>
   </property>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>1021</width>
      <height>101</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ForbiddenCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font: bold 24pt &quot;Microsoft YaHei UI&quot;;
    color: #ffffff;
    background: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 0,
                                 stop: 0 #667eea, stop: 1 #764ba2);
    border-radius: 12px;
    padding: 15px;
    margin: 5px;
}
</string>
    </property>
    <property name="text">
     <string>交 易 完 成</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>530</y>
      <width>631</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="back">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    font: bold 14pt &quot;Microsoft YaHei UI&quot;;
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #ffffff, stop: 1 #f8f9fa);
    border: 2px solid #007bff;
    border-radius: 12px;
    color: #007bff;
    padding: 12px 20px;
    min-height: 20px;
}
QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #007bff, stop: 1 #0056b3);
    color: white;
    border: 2px solid #0056b3;
}
QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #0056b3, stop: 1 #004085);
}
QPushButton:disabled {
    background-color: #e9ecef;
    color: #6c757d;
    border: 2px solid #dee2e6;
}
</string>
       </property>
       <property name="text">
        <string>繼續操作</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="logout">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton {
    font: bold 14pt &quot;Microsoft YaHei UI&quot;;
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #ff6b6b, stop: 1 #ee5a52);
    border: 2px solid #dc3545;
    color: white;
    border-radius: 12px;
    padding: 12px 20px;
    min-height: 20px;
}
QPushButton:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #dc3545, stop: 1 #c82333);
}
</string>
       </property>
       <property name="text">
        <string>登出</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>961</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_type">
       <property name="cursor">
        <cursorShape>ForbiddenCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    font: bold 14pt &quot;Microsoft YaHei UI&quot;;
    color: #495057;
    background: #fffbe6;
    border-radius: 8px;
    padding: 6px 12px;
}
</string>
       </property>
       <property name="text">
        <string>交易類型：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>15</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_amount">
       <property name="cursor">
        <cursorShape>ForbiddenCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    font: bold 14pt &quot;Microsoft YaHei UI&quot;;
    color: #495057;
    background: #fffbe6;
    border-radius: 8px;
    padding: 6px 12px;
}
</string>
       </property>
       <property name="text">
        <string>金額：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>15</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_target">
       <property name="cursor">
        <cursorShape>ForbiddenCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    font: bold 14pt &quot;Microsoft YaHei UI&quot;;
    color: #495057;
    background: #fffbe6;
    border-radius: 8px;
    padding: 6px 12px;
}
</string>
       </property>
       <property name="text">
        <string>對象：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>15</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_balance">
       <property name="cursor">
        <cursorShape>ForbiddenCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    font: bold 14pt &quot;Microsoft YaHei UI&quot;;
    color: #495057;
    background: #fffbe6;
    border-radius: 8px;
    padding: 6px 12px;
}
</string>
       </property>
       <property name="text">
        <string>結餘：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
